app/ui/client/views/app/room.js

```diff
@@ -931,6 +950,7 @@ Template.room.onCreated(function() {
 	this.flexTemplate = new ReactiveVar;
 
 	this.userDetail = new ReactiveVar(FlowRouter.getParam('username'));
+	this.groupDetail = new ReactiveVar('all');
```

Comment:
this changed the default member list options from `Online` to `All`.. is that intentional?

Reply:
It wasn't, I'll patch it.